lib/ice_cube/time_util.rb

```diff
@@ -1,5 +1,7 @@
-require "date"
-require "time"
+require 'date'
+require 'time'
+require 'active_support'
+require 'active_support/core_ext'
```

Comment:
This introduces a new dependency for the gem on `active_support`. The `gemspec` file probably needs to be updated accordingly. A non-Rails Ruby app won't necessarily have active support available so it needs to be listed as an explicit dependency.

Reply:
@jkehres Thanks, I've fixed this now.  These were a holdover from the original commit in 2016 but TimeUtil was refactored since then to no longer need ActiveSupport for time arithmetic.